src/step.cpp

```diff
@@ -15,6 +15,9 @@
 %  Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 */
 
+#include <array>
+#include <iostream>
```

Comment:
Are we using iostream for anything here?

Reply:
Removed.